Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rsmgclient"
-version = "0.1.1"
+version = "0.2.0"
```

Comment:
@antaljanosbenjamin Since this version changes the API, but it's not yet 1.0, I think the best version is 0.2 for now/